pkg/manager/internal.go

```diff
@@ -251,7 +251,7 @@ func (cm *controllerManager) SetFields(i interface{}) error {
 	if _, err := inject.MapperInto(cm.mapper, i); err != nil {
 		return err
 	}
-	if _, err := inject.LoggerInto(log, i); err != nil {
+	if _, err := inject.LoggerInto(cm.logger, i); err != nil {
```

Comment:
What a mess this is, so the `log` variable is:


*  `logf.RuntimeLog.WithName("manager")` where
  -> `logf.RuntimeLog` is `log.Log.WithName("controller-runtime")` where
    -> `log.Log` is `NewDelegatingLogger(NullLogger{})`

And the managers logger is defaulted to `logf.Log`

So I think we should have the `manager`  default `options.Logger` to what is currently the `log` variable instead, right?

ALso, can you check if the `log` variable is used by anything else and if not, remove it and move the construction of its logger into the managers log defaulting?

/ok-to-test

Reply:
Yes, valid point.
Changed the defaulting of the manager's logger to `logf.RuntimeLog.WithName("manager")` and switched the remaining usages of the `log` variable now.
Hope I got you right, PTAL